opentracing-examples/src/main/java/io/vertx/example/opentracing/Gateway.java

```diff
@@ -26,9 +27,9 @@ public static void main(String[] args) throws InterruptedException, ExecutionExc
       .withSampler(samplerConfig)
       .withReporter(reporterConfig);
 
-    Vertx vertx = Vertx.vertx(new VertxOptions()
-      .setTracingOptions(new OpenTracingOptions(config.getTracer())
-      ));
+    Vertx vertx = Vertx.builder()
```

Comment:
Need back-port to 4.x: tracer factory should be able to wrap an existing tracer.

Reply:
https://github.com/eclipse-vertx/vertx-tracing/pull/78